cartoframes/data/clients/bigquery_client.py

```diff
@@ -34,36 +34,55 @@ class BigQueryClient(object):
     def __init__(self, project, credentials):
         self._project = project
         self._credentials = credentials or get_default_credentials()
-        self.client = self._init_client()
```

Comment:
dont you think it would be nice to init the clients in the constructor? If not, in the first usage, is going to fail if I am not wrong

Reply:
Yes, but `self.bq_client, self.gcs_client = self._init_clients()` instead. I'm going to change it

Reply:
302c716